docs/proposals/16-adminkubeconfig-subresource.md

```diff
@@ -96,7 +96,7 @@ status:
         client-key-data: LS0tLS1CRUd...
 ```
 
-The `AdminKubeconfigRequest` feature gate (enabled by default starting from `v1.39`) enables the above mentioned API in the `gardener-apiserver`. The old `{shoot-name}.kubeconfig` is kept, but deprecated and will be removed in the future.
+The `AdminKubeconfigRequest` feature gate (enabled strictly starting from `v1.42`) enables the above mentioned API in the `gardener-apiserver`. The old `{shoot-name}.kubeconfig` is kept, but deprecated and will be removed in the future.
```

Comment:
The proposal says
> The old `{shoot-name}.kubeconfig` is kept, but deprecated and will be removed in the future.

Is this still applicable? If yes, how we follow-up on it? (cc @petersutter )

Reply:
Good point. From what I understood the current plan is to let the shoot owner disable the creation of this admin kubeconfig https://github.com/gardener/gardener/issues/3138